### PR TITLE
Add export for all utils

### PR DESCRIPTION
**Pull request by brettvniekerk**

### CODE REVIEW

**Review by Preston-Harrison — APPROVED** [review]

**Review by nabsta-eth — DISMISSED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ import {
   SECONDS_IN_A_YEAR_BN,
 } from "./constants";
 import { vaultUtils } from "./utils/vault-utils";
+import * as Utils from "./utils";
```
Comment (rellfy) [inline src/index.ts:50]: Lowercase please
```suggestion
import * as utils from "./utils";
```

File: src/index.ts [hunk 2]
```diff
@@ -83,6 +84,7 @@ export {
   getIsKashiApproved,
   signKashiApproval,
   vaultUtils,
+  Utils,
```
Comment (rellfy) [inline src/index.ts]:
```suggestion
  utils,
```

Comment (brettvniekerk) [reply src/index.ts]: done

File: src/utils/index.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  ConvertUtils,
+  FxTokenUtils,
+  GeneralUtils,
+  H2SOUtils,
+  PriceUtils,
+  SushiSwapUtils,
+  TokenListUtils,
+  TokenTradeUtils,
+  VaultUtils,
+};
```
Comment (rellfy) [inline src/utils/index.ts:27]: All the `Utils` suffixes here are redundant. Please make the initial lowercase too,
e.g. `allowance`
So we access as `sdk.utils.allowance`

Comment (brettvniekerk) [reply src/utils/index.ts]: done

**Review by nabsta-eth — DISMISSED** [review]

File: src/utils/index.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import * as allowance from "./allowance-utils";
+import * as collateral from "./collateral-utils";
+import * as contract from "./contract-utils";
+import * as convert from "./convert-utils";
+import * as fxToken from "./fxToken-utils";
+import * as general from "./general-utils";
+import * as h2so from "./h2so-utils";
```
Comment (rellfy) [inline src/utils/index.ts:7]: If you look at this file, it is all deprecated: https://github.com/handle-fi/handle-sdk/blob/master/src/utils/h2so-utils.ts
Please remove this one

Comment (brettvniekerk) [reply src/utils/index.ts:7]: will do

**Review by rellfy — APPROVED** [review]

**Review by nabsta-eth — APPROVED** [review]

**Pull request merged**